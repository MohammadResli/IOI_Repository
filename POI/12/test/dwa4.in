50
36 7 21 30 36 34 40 50 12 47 33 13 44 41 25 24 31 45 49 15 35 42 28 10 39 22 11 17 19 29 18 27 23 32 37 16 5
31 5 18 29 14 27 42 49 50 10 20 41 19 17 21 30 33 40 45 24 28 44 35 32 8 22 23 47 31 48 15 4
29 17 28 26 46 43 35 49 42 29 37 9 15 40 18 12 38 23 11 47 27 36 45 22 30 50 19 6 25 14
37 48 30 11 14 38 20 39 35 18 41 8 12 43 15 10 2 46 24 49 16 42 9 40 44 17 32 25 37 31 29 5 28 19 26 34 23 6
35 7 11 1 26 10 30 28 29 21 4 33 40 25 14 48 24 12 17 18 13 2 19 15 49 46 44 6 32 42 22 43 8 20 23 39
40 37 44 39 27 11 33 32 13 38 22 4 30 18 8 29 19 20 24 43 40 3 28 5 15 25 34 16 23 9 7 35 10 31 42 21 17 48 41 12 47
29 35 32 41 8 45 10 26 34 43 11 22 36 1 50 44 28 13 33 19 6 49 5 25 40 14 46 21 47 38
32 33 28 5 15 30 43 46 17 27 39 38 4 29 6 26 44 20 49 19 41 18 7 16 21 13 12 22 24 23 11 2 9
34 39 33 45 3 14 6 29 35 43 46 41 23 37 50 13 32 31 34 36 44 10 4 40 19 25 15 22 24 38 8 17 49 12 30
36 40 46 9 5 21 1 18 34 19 30 12 26 44 6 25 45 15 7 17 33 47 36 27 50 32 4 2 14 16 24 22 48 13 39 49 23
38 27 23 31 45 1 14 24 42 34 5 38 12 17 35 15 46 37 41 13 39 49 3 16 22 44 48 4 7 40 26 18 6 8 29 33 30 25 32
36 48 36 24 30 47 32 3 27 19 45 20 26 29 31 5 42 6 10 21 50 44 1 28 49 25 11 46 40 9 8 34 13 33 38 15 4
33 49 30 33 46 11 34 24 6 10 17 5 23 28 43 7 48 9 8 16 47 37 32 26 1 18 21 14 12 42 20 40 38 36
33 9 11 16 44 25 27 33 4 22 46 5 28 43 35 23 26 47 48 3 30 13 29 17 21 2 40 20 10 7 41 38 45 24
34 30 27 49 25 32 2 48 17 1 46 11 47 26 5 6 3 44 23 16 9 12 37 38 50 8 45 24 31 40 39 10 43 4 20
32 22 38 50 46 14 24 15 27 41 1 4 42 11 49 21 37 6 8 32 20 30 10 43 26 47 29 48 18 23 44 13 33
34 1 46 25 14 15 31 8 10 45 23 42 3 28 38 6 29 5 9 36 13 49 48 19 18 33 4 39 20 22 24 34 21 11 2
35 30 41 23 49 6 5 24 33 10 3 50 47 17 32 45 28 25 39 46 8 27 36 13 29 40 20 11 42 1 16 4 35 38 43 2
33 10 5 4 9 6 30 48 7 8 34 25 32 35 33 26 23 36 41 44 42 12 43 47 37 2 1 3 46 27 28 49 17 24
31 17 42 34 6 27 36 4 15 40 45 48 18 44 8 28 2 35 22 39 12 5 47 33 13 16 46 41 14 26 32 38
36 30 45 39 47 34 46 35 14 25 12 2 48 6 37 13 44 49 28 7 8 43 24 36 50 16 5 42 38 17 31 33 29 26 10 22 1
33 37 14 17 20 41 33 39 38 3 11 6 46 10 50 16 2 40 9 7 44 8 21 34 47 23 28 49 25 5 27 42 1 43
34 4 9 25 42 2 6 22 26 15 1 48 8 27 40 31 19 5 35 17 18 32 13 10 28 11 38 41 39 14 30 47 46 16 3
32 2 6 49 40 16 18 21 39 38 15 9 4 26 34 37 43 11 32 17 30 5 19 28 12 8 1 14 10 47 29 35 13
33 23 29 1 50 11 14 4 15 19 5 18 6 33 47 28 40 9 48 30 45 3 42 12 26 31 37 46 22 10 17 21 7 43
37 3 23 7 20 5 14 10 43 40 49 34 13 19 8 27 31 28 24 46 25 29 35 48 12 41 37 16 4 15 38 30 50 42 33 11 21 45
29 50 3 14 32 11 22 36 29 8 23 10 20 15 30 49 42 31 48 16 18 1 33 2 19 47 26 12 43 6
34 40 26 12 2 20 7 39 31 8 36 14 25 4 50 5 3 45 6 29 18 1 41 19 38 17 24 30 13 21 23 22 48 49 37
35 38 26 4 33 44 39 50 16 11 8 24 14 32 1 12 9 6 40 48 3 21 49 45 28 18 27 2 25 41 5 17 43 46 37 35
37 6 14 25 18 15 12 48 27 35 5 38 39 28 3 47 4 11 41 2 16 9 32 49 46 21 43 24 26 19 23 31 36 13 40 10 1 8
30 12 9 49 34 15 4 45 1 17 42 41 50 48 25 36 6 30 27 39 2 11 21 38 26 46 47 43 40 23 28
35 15 19 30 16 1 44 5 11 24 33 36 20 29 18 10 7 42 2 35 13 23 48 9 4 46 41 43 12 50 37 27 47 40 49 6
35 20 16 41 32 48 21 47 5 8 34 43 12 19 29 25 50 13 11 36 1 22 35 40 17 14 27 18 9 6 10 39 7 2 26 42
30 50 11 42 9 48 7 40 49 24 33 13 26 19 31 39 4 17 22 43 1 6 21 12 35 20 36 10 47 44 46
30 45 30 50 18 21 24 20 43 38 23 1 4 2 19 41 32 26 29 33 40 11 47 34 3 7 6 48 46 9 14
30 31 17 34 18 38 32 49 37 50 44 12 42 3 10 1 30 19 27 21 45 20 48 7 13 33 39 43 40 28 9
29 19 39 29 47 16 24 36 22 1 48 26 38 13 42 4 6 11 15 21 32 50 25 44 3 45 28 43 49 9
33 8 13 30 15 17 12 11 21 16 42 4 14 23 3 6 22 7 36 9 35 28 49 50 20 29 31 24 26 43 37 39 41 18
31 29 46 45 18 38 24 21 4 17 44 1 10 8 22 37 9 28 30 34 31 23 36 11 41 33 15 47 40 5 20 6
39 36 30 45 44 18 4 35 1 23 49 24 12 25 13 34 29 7 11 39 2 15 31 48 42 26 9 5 22 43 50 10 33 32 14 6 46 28 20 3
30 32 48 16 20 2 31 1 8 19 23 4 22 33 46 9 39 29 35 45 14 18 44 6 11 26 38 7 42 28 30
36 21 36 5 23 32 22 4 37 44 1 40 12 38 2 46 33 3 31 19 26 27 11 13 34 43 16 6 47 50 18 45 41 17 20 49 25
35 13 25 49 3 22 33 21 7 38 24 31 15 5 34 35 26 4 16 9 14 44 27 19 29 8 37 18 36 30 40 45 47 32 42 6
32 1 29 14 11 34 2 4 10 20 39 46 36 47 7 9 12 50 5 8 42 22 21 37 16 19 40 41 45 32 6 15 43
30 21 48 35 2 40 10 20 28 37 41 11 9 15 43 31 12 1 3 39 18 25 7 44 14 29 50 36 42 26 17
36 30 40 13 17 16 4 20 8 14 21 42 22 9 41 35 44 25 32 10 5 48 12 11 50 31 26 34 15 29 49 19 7 39 18 23 3
33 14 21 31 27 6 34 37 3 16 12 7 39 18 1 30 24 10 48 42 50 23 22 2 25 15 44 20 32 35 43 33 13 19
36 23 21 50 5 14 28 47 49 31 35 19 41 45 25 26 32 4 2 33 6 16 34 20 10 15 46 12 40 11 36 29 37 30 17 13 27
36 11 38 13 7 16 28 46 22 19 18 48 43 17 26 15 34 1 37 27 9 4 29 21 42 8 32 31 2 5 30 12 24 3 40 10 36
32 26 12 29 2 3 28 42 1 21 44 18 22 48 16 7 10 33 40 35 31 15 25 38 9 46 27 32 47 45 36 37 34
