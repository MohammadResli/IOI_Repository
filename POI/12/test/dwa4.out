20
1 2 5 6 13 14 18 19 20 25 33 34 36 37 41 43 46 48 49 50 
